specification/attestation/resource-manager/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -77,17 +77,16 @@
         },
         "parameters": [
           {
-            "$ref": "#/parameters/SubscriptionIdParameter"
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "#/parameters/ResourceGroupNameParameter"
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
           },
           {
             "name": "providerName",
             "in": "path",
             "required": true,
             "type": "string",
-            "pattern": "^[a-z0-9]{3,24}$",
```

Comment:
It seems that this is the real change. Since this is a preview version and this will not bring breaking change. I think it's ok.

Reply:
The readme.md file is already generated and published in the repo under attestation. This pull request is only a update for the existing attestation.jaon file.